### ISSUE TITLE
Banner cut off on mobile

### ISSUE THREAD

**Issue by ldub** [ekafyi/gatsby-theme-musician#3]
You can see that the banner gets cut off on mobile (horizontal cut off point circled in red below):

<img width="404" alt="undefined" src="https://user-images.githubusercontent.com/3114081/65216889-41af3980-da67-11e9-8401-d77f44088894.png">

On your demo site, it doesnt look that bad because the banner's background is black. However, if the banner doesn't have a black background then it really sticks out. Is there any way to make it stretch/cover the banner vertically to fit the background?

**Comment by arichardmusic** [comment]
Yes, experiencing the same problem with the https://arichardmusic.netlify.com site that's in development. Very pronounced in this case because of the red photo background.

**Comment by ldub** [comment]
@arichardmusic as a quick stopgap, commenting out this line:

https://github.com/ekafyi/gatsby-theme-musician/blob/ada534c670954ca118827cdd7e415e75c6831923/theme/src/gatsby-plugin-theme-ui/componentStyles.js#L11

helps with this problem. it does not make the background stretch, but instead makes the black portion smaller.

**Comment by charles-hood** [comment]
> @arichardmusic as a quick stopgap, commenting out this line:
> 
> https://github.com/ekafyi/gatsby-theme-musician/blob/ada534c670954ca118827cdd7e415e75c6831923/theme/src/gatsby-plugin-theme-ui/componentStyles.js#L11
> 
> helps with this problem. it does not make the background stretch, but instead makes the black portion smaller.

Is there a way to shadow this component to accomplish this?
